transformers/DsChiefTransformer.js

```diff
@@ -15,10 +19,45 @@ module.exports = (address, nameSuffix = '') => ({
   },
 });
 
+const vdQuery = `SELECT *
+    FROM dschief.lock
+    WHERE tx_id = $1
+    AND log_index = $2`;
+
 const handlers = {
   "free(uint256)": async (services, { log, note }) => {
     const tx = await getTxByIdOrDie(services, log.tx_id);
 
+    //get delegate event
+    try {
+      //TODO: use RetryProvider
```

Comment:
Is this TODO something we need to update before merging?

Reply:
Actually yeah, I just made the change to use the provider in spock instead.